tests/unit/prediction/upc_image/test_upc_image.py

```diff
@@ -0,0 +1,28 @@
+import pathlib
+
+import cv2
+import pytest
+
+from robotoff.prediction.upc_image import UPCImageType, find_image_is_upc
+
+data_dir = pathlib.Path(__file__).parent / "data"
+
+
+@pytest.mark.parametrize(
+    "imgpath, expected",
+    [
+        (data_dir / "upc1.jpg", UPCImageType.UPC_IMAGE),
+        (data_dir / "upc2.jpg", UPCImageType.UPC_IMAGE),
+        (data_dir / "upc3.jpg", UPCImageType.UPC_IMAGE),
+        (data_dir / "small_upc1.jpg", UPCImageType.SMALL_UPC),
+        (data_dir / "small_upc2.jpg", UPCImageType.SMALL_UPC),
+        (data_dir / "small_upc3.jpg", UPCImageType.SMALL_UPC),
+        (data_dir / "no_upc1.jpg", UPCImageType.NO_UPC),
+        (data_dir / "no_upc2.jpg", UPCImageType.NO_UPC),
+        (data_dir / "no_upc3.jpg", UPCImageType.NO_UPC),
```

Comment:
It looks like you've added the images using standard git commands and not git LFS, right? (https://git-lfs.com/)
Images are quite heavy, and we don't want to store them directly in the repository.

Reply:
Got it, this should be fixed now.